acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -6,6 +6,9 @@
 
 # This file contains general test helper methods for pxp-agent acceptance tests
 
+# The standard path for git checkouts is where pcp-broker will be
+GIT_CLONE_FOLDER = '/opt/puppet-git-repos'
```

Comment:
I'm not happy with this being defined here. I've created PCP-298 to refactor this lib - possibly at this point pcp-broker deserves its own helper file